docs/quickstart.rst

```diff
@@ -131,6 +131,23 @@ Jira Webhook
     follow the instructions in the ``Testing on a Machine Not Accessible to Jira`` section of the
     debugging guide.
 
+Jira Permissions
+================
+In order for the Jira Bridge to be able to create and update issues in Jira, the Jira user associated with the Bridge
+will need to be given certain permissions. Depending on your current Jira permission rules, you may not need to make
+any changes here. But if you would like to restrict the user to only the required permissions, you will need to add
```

Comment:
The only thing I'd suggest is to change this to "the Jira user associated with the Bridge
needs to have certain permissions". Less wordy, and less passive voice.